frontend/src/routes/Applications/CreateApplication/Subscription/controlData/ControlDataPlacement.js

```diff
@@ -184,7 +180,8 @@ export const reverseExistingRule = (control, templateObject) => {
 }
 export const reverseExistingRuleName = (control, templateObject) => {
     const active = _.get(templateObject, getSourcePath('Subscription[0].spec.placement.placementRef.name'))
-    if (active && control.type !== 'hidden' && control.active === undefined) {
+
+    if (active && control.active === undefined) {
         const { groupControlData } = control
         const selectedRuleName = groupControlData.find(({ id }) => id === 'selectedRuleName')
         selectedRuleName.active = active.$v
```

Comment:
Can we try eliminating this function and just using `reverseExistingRule` for both the dropdown and the hidden control?